### DISCUSSION TITLE
Tailwind is❣️

### DISCUSSION THREAD

**Discussion by ImOmkar** [tailwindlabs/tailwindcss#3219] [Kind words]
👇 Designed using Tailwind CSS. 🚀
https://theframeartindia.com
http://thelekhani.com

**Comment by simonswiss** [comment]
Looking great @ImOmkar!

Love the rotated testimonials ✨

![image](https://user-images.githubusercontent.com/485747/103492030-5d1c9e80-4e7c-11eb-8601-bd651d8a954b.png)
